integration-app/src/test/java/uk/gov/crowncommercial/esourcing/integration/api/ApiExceptionHandlerRollbarDisabledIT.java

```diff
@@ -78,12 +82,12 @@ public static void setDynamicProperties(DynamicPropertyRegistry registry) {
   public void salesforce_throwsNullPointerException_expectInternalServerErrorAndRollbarSend()
       throws Exception {
     /* mock the service call */
-    when(tenderApiService.createProcurementCase(any(ProjectTender.class))).thenThrow(new NullPointerException(
+    when(tenderApiService.createCase(any(ProjectTender.class))).thenThrow(new NullPointerException(
         "Thrown as part of getTenderById_throwsNullPointerException_expectInternalServerError"));
 
     /* "call" the REST API */
     MvcResult mvcResult = mockMvc
-        .perform(MockMvcRequestBuilders.post(CCS_API_BASE_PATH + "/tenders/ProcurementProjects/salesforce")
+        .perform(MockMvcRequestBuilders.put(CCS_API_BASE_PATH + "/tenders/ProcurementProjects/salesforce")
```

Comment:
Maybe worth creating a BASE_PATH that include the /tenders/ProcurementProjects to simplify changes in the future if/when the path changes.

Reply:
done